<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Classcal_Game_Setting</class>
 <widget class="QWidget" name="Classcal_Game_Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>525</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="HeadContainer">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="FoodContainer" stretch="1,5,1">
      <item>
       <widget class="QLabel" name="setFoodnumLabel">
        <property name="text">
         <string>最大食物数量：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="setFoodnum">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="showFoodnum">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>值：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="SpeedContainer" stretch="1,5,1">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="setSpeedLable">
        <property name="text">
         <string>蛇多少毫秒移动一格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="setSpeed">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="showSpeed">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>值：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Label">
        <property name="text">
         <string>设置地图行数：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Label_2">
        <property name="text">
         <string>设置地图列数：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="setRow">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="setCol">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Label_3">
        <property name="text">
         <string>蛇头撞到身体行为：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_4">
        <property name="text">
         <string>蛇撞到墙后：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="setMode">
        <item>
         <property name="text">
          <string>无发生任何事</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>蛇身缩短一格</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>死亡</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="throughWall">
        <item>
         <property name="text">
          <string>穿到地图另一边</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>死亡</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Label_5">
        <property name="text">
         <string>蛇身渐变色颜色设置：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="showColor_1">
        <property name="text">
         <string>渐变色1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="showColor_2">
        <property name="text">
         <string>渐变色2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="setColor_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
        </property>
        <property name="text">
         <string>设置渐变色1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="setColor_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
        </property>
        <property name="text">
         <string>设置渐变色2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>100</number>
      </property>
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <item>
       <widget class="QPushButton" name="SaveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
        </property>
        <property name="text">
         <string>保存更改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BackButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(0, 255, 255, 69), stop:0.375 rgba(0, 73, 255, 69), stop:0.421296 rgba(0, 141, 255, 145), stop:0.43318 rgba(102, 168, 188, 171), stop:0.437788 rgba(85, 207, 195, 181), stop:0.442396 rgba(67, 225, 225, 192), stop:0.447005 rgba(56, 222, 244, 202), stop:0.45 rgba(0, 255, 238, 208), stop:0.456221 rgba(70, 253, 229, 191), stop:0.460829 rgba(27, 225, 251, 178), stop:0.470046 rgba(94, 209, 247, 151), stop:0.477581 rgba(71, 222, 255, 130), stop:0.520737 rgba(71, 197, 255, 130), stop:0.55 rgba(0, 234, 255, 255), stop:0.576037 rgba(0, 234, 255, 130), stop:0.625 rgba(36, 0, 255, 69), stop:1 rgba(0, 255, 255, 69));</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
